minimessage-plugin/src/main/java/me/minidigger/minimessage/plugin/MiniMessagePlugin.java

```diff
@@ -18,19 +24,70 @@
 
     private BukkitPlatform platform;
 
+    private BukkitTask task;
+    private int progress = 0;
+
     @Override
     public void onEnable() {
         platform = BukkitPlatform.of(this);
     }
 
     @Override
-    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] a) {
+        String args = String.join(" ", a);
+        switch (command.getName()) {
+            case "minimessage":
+                minimessage(sender, args);
+                break;
+            case "minibossbar":
+                minibossbar(sender);
+                break;
+            case "minispam":
+                minispam(sender);
+                break;
+            default:
+                return false;
+        }
+        return true;
+    }
+
+    private void minispam(CommandSender sender) {
+        BossBar bar = BossBar.of(getWithProgress(0), 1, BossBar.Color.BLUE, BossBar.Overlay.PROGRESS);
+        if (task != null) {
+            task.cancel();
+        }
+        Audience audience = platform.audience(sender);
+        task = Bukkit.getScheduler().runTaskTimer(this, () -> {
+            Component message = getWithProgress(progress++);
+            bar.name(message);
+            audience.sendActionBar(message);
+            audience.sendMessage(message);
+            audience.showTitle(Title.of(message, message, Duration.ZERO, Duration.of(20, ChronoUnit.MILLIS), Duration.ZERO));
```

Comment:
Durations are converted to ticks -- so any duration less than 50ms will result in undefined behaviour.

Reply:
I tried to do one tick there actually. Does adventure have a chronounit constant for something for ticks?